vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/node.go

```diff
@@ -718,27 +718,29 @@ var _ = DescribeSanity("Node Service", func(sc *SanityContext) {
 		}
 		// NodeStageVolume
 		if nodeStageSupported {
-			By("node staging volume")
```

Comment:
Better define two separate test cases: one which does idempotent calls with a configurable number of repetitions (at least two, perhaps better more to stress the driver a bit - 10?) and one which always invokes each function only once. The body of the tests can be the same function with a repetition parameter.

Reply:
something like this? (pushed new version)